src/sentry/static/sentry/app/views/organizationStats/index.jsx

```diff
@@ -1,4 +1,4 @@
-import $ from 'jquery';
+import _ from 'lodash';
```

Comment:
This wasn't really necessary, it got pulled with some other changes I was exploring (using jQuery's slim build which doesn't have `$.extend`) ... I can revert.

Reply:
../